src/platform/packages/shared/kbn-alerts-as-data-utils/src/schemas/generated/observability_apm_schema.ts

```diff
@@ -87,6 +87,7 @@ const ObservabilityApmAlertOptional = rt.partial({
       value: schemaStringArray,
     })
   ),
+  'kibana.alert.grouping': schemaUnknown,
```

Comment:
just curious, are these auto-generated changes since the files are in "generated" folder?

Reply:
Yes, it is auto-generated.